tests/test/Foundry.t.sol

```diff
@@ -27,4 +27,28 @@ contract FoundryTest is Test {
         counter2.set(x);
         assertEq(counter2.cnt(), x);
     }
+
+    function testEtchConcrete() public {
+        vm.etch(address(0x42), hex"60425f526001601ff3");
+        (bool success, bytes memory retval) = address(0x42).call("");
+
+        assertTrue(success);
+        assertEq(retval.length, 1);
+        assertEq(uint256(uint8(retval[0])), 0x42);
```

Comment:
just curious, what's the meaning of the return value here?  i mean, what does the code do?

Reply:
it just returns 0x42, just wanted some "proof" that the code ran and returned the expected value

Reply:
i see.  i was wondering whether it would mean the address `0x42` used in `vm.etch()`, or not.